src/templates/understandTopic.html

```diff
@@ -1,23 +1,23 @@
 <div class="ms-5 me-5">
     <h1 class="bolded-blue" data-i18n="moral-page-1-2">###</h1>
-    <h2 class="bolded-blue" data-i18n="moral-topic-understand-{{topic}}">###</h1>
```

Comment:
why change font size here?
please make sure this change will not influence other survey and future survey